cleanrl/ppo_mujoco_envpool_xla_jax.py

```diff
@@ -0,0 +1,679 @@
+# docs and experiment results can be found at https://docs.cleanrl.dev/rl-algorithms/ppo/#ppo_atari_envpool_xla_jaxpy
```

Comment:
I suggest renaming the file to `ppo_continuous_action_envpool_xla_jax_scan.py` to be consistent with the naming convention of other variants. We might also be able to apply this file on the DMControl envs from envpool (https://envpool.readthedocs.io/en/latest/env/dm_control.html)